comment-syntax.md

```diff
@@ -84,20 +127,33 @@ In TiDB, you can also use another version:
 SELECT STRAIGHT_JOIN col1 FROM table1,table2 WHERE ...
 ```
 
+<<<<<<< HEAD
 If the server version number is specified in the comment, for example, `/*!50110 KEY_BLOCK_SIZE=1024 */`, in MySQL it means that the contents in this comment is processed only when the MySQL version is or higher than 5.1.10. But in TiDB, the version number does not work and all contents in the comment are processed.
+=======
+If the server version number is specified in the comment, for example, `/*!50110 KEY_BLOCK_SIZE=1024 */`, in MySQL it means that the contents in this comment are processed only when the MySQL version is or higher than 5.1.10. But in TiDB, the MySQL version number does not work and all contents in the comment are processed. 
+>>>>>>> 0af9946... comment-syntax: add TiDB-specific comment syntax (#3238)
 
-Another type of comment is specially treated as the Hint optimizer:
+## TiDB specific comment syntax
 
-```
-SELECT /*+ hint */ FROM ...;
-```
+TiDB has its own comment syntax (that is, TiDB specific comment syntax), which can be divided into the following two types:
 
-Since Hint is involved in comments like `/*+ xxx */`, the MySQL client clears the comment by default in versions earlier than 5.7.7. To use Hint in those earlier versions, add the `--comments` option when you start the client. For example:
+* `/*T! Specific code */`: This syntax can only be parsed and executed by TiDB, and be ignored in other databases.
+* `/*T![feature_id] Specific code */`: This syntax is used to ensure compatibility between different versions of TiDB. TiDB can parse the SQL fragment in this comment only if it implements the corresponding feature of `feature_id` in the current version. For example, as the `AUTO_RANDOM` feature is introduced in v3.1.1, this version of TiDB can parse `/*T![auto_rand] auto_random */` into `auto_random`. Because the `AUTO_RANDOM` feature is not implemented in v3.0.0, the SQL statement fragment above is ignored.
```

Comment:
Please remove changes from https://github.com/pingcap/docs-cn/pull/3477/files to align the English doc https://docs.pingcap.com/zh/tidb/v3.0/comment-syntax. @ireneontheway